lib/deploy/index.js

```diff
@@ -79,6 +80,10 @@ var deploy = function(targetNames, options) {
       origin: api.uploadOrigin
     });
   }).then(function(res) {
+    if (_.get(options, 'config.notes.rules')) {
```

Comment:
I'm looking to see that it's set to something, not just that the key exists. If the code did `tihs.notes.rules = null` somewhere a `_.has` would still report it. I'd be willing to wrap in a `_.isString` if you'd prefer.